nise/generators/ocp/ocp_generator.py

```diff
@@ -88,20 +89,53 @@
     "namespace",
     "namespace_labels",
 )
+OCP_ROS_USAGE_COLUMN = (
+    "interval_start",
+    "interval_end",
+    "namespace",
+    "node",
+    "resource_id",
+    "pod",
+    "deployment_name",
+    "image_name",
```

Comment:
`deployment_name, image_name` are not collected by the cost-management-metrics-operator.

Reply:
Right but I am adding additionally for the pod metadata.

Reply:
Nise generates reports that mimic reports generated by the operator. So the files that are generated here should contain the same information generated by the operator. `deployment_name` and `image_name` are not gathered by the operator and will not appear in the operator reports. So they should not be included here because there will be a mismatch between Nise and the operator.